src/main/java/com/siit/hospital_manager/service/EmailSender.java

```diff
@@ -0,0 +1,23 @@
+package com.siit.hospital_manager.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EmailSender {
+
+    @Autowired
+    private JavaMailSender javaMailSender;
+
+    public void sendAppointmentConfirmationEmail(String to, String subject, String body) {
+        SimpleMailMessage message = new SimpleMailMessage();
+        message.setTo(to);
```

Comment:
to could be named more suggestive, also null & format checked. (eg: recepient, mailAddress, etc)